sycl/include/syclcompat/device.hpp

```diff
@@ -444,6 +444,10 @@ class device_ext : public sycl::device {
     return get_device_info().get_local_mem_size();
   }
 
+  int get_max_pitch() const { return INT_MAX; }
```

Comment:
I think it may be preferable to use `numeric_limits<int>::max()` from the C++ `<limits>` library instead which is equivalent to `INT_MAX` from `<climits>`.

Also, while this compiles fine due to limits probably being included indirectly from another include, it is better to include the library/headers at the top of this file explicitly.